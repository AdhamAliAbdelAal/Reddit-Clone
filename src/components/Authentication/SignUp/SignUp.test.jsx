// import {
//   render, screen,
// } from '@testing-library/react';
// import '@testing-library/jest-dom';
// import App from '../../../App';

// // Render Login Page
// it('should render Signup page', () => {
//   window.history.pushState({}, '', '/register');
//   render(<App />);
//   expect(screen.getByTestId('register-test')).toBeInTheDocument();
// });
