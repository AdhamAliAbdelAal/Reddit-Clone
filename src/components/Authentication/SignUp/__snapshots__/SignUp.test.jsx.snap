// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigUp test snapshot 1`] = `
<div
  className="MuiBox-root css-pdterq"
  data-testid="signup-test"
>
  <div
    className="MuiBox-root css-1i4eqyw"
  />
  <div
    className="MuiBox-root css-rjktpo"
    data-testid="signup-email-test"
    mnwidth="280px"
    mxwidth="280px"
  >
    <div
      className="MuiBox-root css-eyuwdo"
    >
      <img
        alt="Reddit"
        src="https://www.redditstatic.com/accountmanager/18e257d5fdea817c0f12cccf8867d930.svg"
        width="40px"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-1sg842v-MuiTypography-root"
      >
        Welcome back!
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1 css-g041yz-MuiTypography-root"
      >
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph css-aeviwr-MuiTypography-root"
        >
          You are already logged in and will be redirected back to Reddit shortly.
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph css-aeviwr-MuiTypography-root"
        >
          If you are not redirected automatically, follow
           
          <a
            className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-rfhsen-MuiTypography-root-MuiLink-root"
            href="#"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
          >
            this link
          </a>
          .
        </p>
      </p>
    </div>
  </div>
</div>
`;
